build.gradle

```diff
@@ -90,6 +92,8 @@ ext.externalDependency = [
   "mockito": "org.mockito:mockito-core:1.10.19",
   "salesforceWsc": "com.force.api:force-wsc:29.0.0",
   "salesforcePartner": "com.force.api:force-partner-api:29.0.0",
+  "scala": "org.scala-lang:scala-library:2.10.3",
+  "yammerCore": "com.yammer.metrics:metrics-core:2.1.5",
```

Comment:
This has been replaced by https://dropwizard.github.io/metrics, which is already being used in gobblin.